requirements_base.txt

```diff
@@ -4,7 +4,6 @@ h5py==2.6.0
 jupyter==1.0.0
 numpy==1.11.0
 Pillow==3.2.0
-pyosmium==2.6.0
```

Comment:
You won't be able to add the includes flag via requirements.txt (and even if you did, it probably wouldn't be good, since it takes a hardcoded path), which is why I'm fine if we don't include pyosmium.